src/runtime/runtime_internal.h

```diff
@@ -49,6 +49,9 @@ typedef int32_t intptr_t;
 #define STDOUT_FILENO 1
 #define STDERR_FILENO 2
 
+#define O_RDONLY 0
```

Comment:
I don't disagree, but I'd rather do this in a separate change. I'm pretty sure I moved O_RDONLY from another file in the runtime (since I needed it in another place). If not O_RDONLY, then one of the other linux syscall-ish flags.